src/fsharp/lexhelp.fsi

```diff
@@ -51,7 +52,7 @@ val internal digit : char -> int32
 val internal hexdigit : char -> int32
 val internal unicodeGraphShort : string -> uint16
 val internal hexGraphShort : string -> uint16
-val internal unicodeGraphLong : string -> uint16 option * uint16
+val internal unicodeGraphLong : string -> (uint16 option * uint16) option
```

Comment:
Instead of option of option we might want to create a new type which represents the semantics better.